sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -107,6 +109,7 @@ case class CatalogTablePartition(
     if (parameters.nonEmpty) {
       map.put("Partition Parameters", s"{${parameters.map(p => p._1 + "=" + p._2).mkString(", ")}}")
     }
+    stats.foreach(s => map.put("Partition Statistics", s.simpleString))
```

Comment:
@gatorsmile , this code doesn't make partition stats appear in the output of DESC command. It only adds stats into to CatalogTablePartition.toString output (similar to CatalogTable.toString). Do you still want me to add some tests for this functionality?